.travis.yml

```diff
@@ -0,0 +1,18 @@
+if: branch = master and type IN (pull_request)
+language: cpp
+services:
+  - docker
+env:
+before_install:
```

Comment:
If we were able to use the regular docker for this (see https://github.com/telefonicaid/fiware-orion/pull/3192/files#r191405706) this before_install would be unneded.

Reply:
i divided  1 command to 2 (before_script and script) because of usability (ux). Travis hide automatically all steps except script itself, so u can see main result of test faster, without scrolling build step.

Reply:
And we have to use CI Dockerfile, not regular, because of time limits in travis.

Reply:
How much time is the travis time limit?

Reply:
https://docs.travis-ci.com/user/customizing-the-build/#Build-Timeouts, 50 min. Building dep in original Dockerfile tooks additional 15 min.

Reply:
NTC